src/main/java/mat/DTO/fhirconversion/ConversionOutcome.java

```diff
@@ -0,0 +1,26 @@
+package mat.DTO.fhirconversion;
+
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+public enum ConversionOutcome {
```

Comment:
I would make this a string instead of an enum. It makes it more flexible on the MS side this way. They can add whatever we want and we don't have to update this enum every time they do it because it breaks.